content/en/borealis/quick-start/borealis-canary-get-started.md

```diff
@@ -0,0 +1,189 @@
+
+This quick start assumes that you completed the prior two quick starts that taught you how to register a cluster with Borealis and how to deploy an app with the CLI.
+
+To complete this quick start, you need the following:
+
+- Access to a Kubernetes cluster where you can install the Remote Network Agent (RNA). This cluster acts as the deployment target for the sample app. You can reuse the clusters from the previous quick starts if you want. Or stand up new ones.
+- A Prometheus instance set up to monitor your Kubernetes clusters. Keep the following in mind:
+
+  - Armory recommends and the example queries require the flag `"prometheus.io/scrape": "true"`. (This is the default behavior.) This flag instructs Prometheus to collect all Kubernetes annotations, which allow you to reference the annotations that Borealis injects as part of your query.
+   
+   If you install Prometheus with Helm, this example command includes the required flag:
+
+   ```yaml
+   helm upgrade --install prometheus prometheus-community/kube-prometheus-stack --set 'kube-state-metrics.metricAnnotationsAllowList[0]=pods=[*]'
+   ```
+
+  - It is either accessible by the public internet or you have installed the Remote Network Agent (RNA) in the same cluster.
+
+   For information about how to install Prometheus, see the the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/installation/). 
+
+
+## Add your metrics provider
+
+Borealis can run queries against metrics providers that you add. The results are examined as part of canary analysis steps in the deploy file.
+
+1. In the **Configuration UI**, go to [**Canary Analysis > Integrations**](https://console.cloud.armory.io/configuration/metric-source-integrations/).
+2. Select **New Integration.
+
+   The examples in this guide use Prometheus as the metrics provider.
+
+3. Complete the wizard:
+  
+   The parameters you need to provide depend on the metrics provider you choose. For more information, see [Canary Analysis Integrations]({{< ref "borealis-configuration-ui#integrations" >}}).
+
+   The following fields are for a Prometheus integration:
+
+   - **Type**: (Required) Your metrics provider. This example uses Prometheus. The form options change based on your provider. For more information, see [Canary Analysis Integrations]({{< ref "borealis-configuration-ui#integrations" >}}).
+   - **Name**: (Required) A descriptive name for your metrics provider, such as the environment it monitors. You use this name in places such as your deploy file when you want to configure canary analysis as part of your deployment strategy.
+   - **Base URL**: (Required) The base URL for your Prometheus instance. If Prometheus runs in the same cluster as the RNA and is exposed using HTTP on port 9090 through a service named `prometheus` in the namespace `prometheus`, then use `http://prometheus.prometheus:9090`. (This can be a private DNS only if the RNA is installed in the same cluster as the Prometheus instance.)
```

Comment:
this is confusing for a user, as to which one is the Service name and which is the namespace name in the Base URL. if my prometheus namespace is 'monitoring' with a service named 'prometheus' then what is the URL?